src/app/payment/services/payment.service.ts

```diff
@@ -175,7 +175,7 @@ const paymentService = {
   async handlePaymentTeams(priceId: string, quantity: number, mode: StripeSessionMode): Promise<void> {
     const mnemonicTeam = generateMnemonic(256);
     const encMnemonicTeam = await encryptPGP(mnemonicTeam);
-    const codMnemonicTeam = Buffer.from(encMnemonicTeam).toString('base64');
+    const codMnemonicTeam = btoa(encMnemonicTeam as string);
```

Comment:
I hope it's the same. Update of dependencies ( types/node ?) caused this line to fail build

Reply:
This is deprecated code, you can directly remove this

Reply:
@sg-gs  This function is used here https://github.com/internxt/drive-web/blob/8bf9912b4c550b98b6282786af6316e7705dd2e2/src/app/store/slices/payment/index.ts#L64 Should I remove try or all export as well?

Reply:
Done